AxonDeepSeg/train_network.py

```diff
@@ -54,588 +58,178 @@ def train_model(path_trainingset, path_model, config, path_model_init=None,
     dropout = config["dropout"]
     weighted_cost = config["weighted_cost-activate"]
     batch_size_training = config["batch_size"]
-    batch_size_validation = 8
+    batch_size = config["batch_size"]
+    epochs = 5
```

Comment:
Training typically ran for much more than 5 epochs in the past - any reason why this number (5) is hardcoded here?